UPGRADING.md

```diff
@@ -0,0 +1,20 @@
+# Upgrade Notes
+
+## 1.14 -> 1.15
+
+### Cluster Security Group
+
+Existing clusters will be using separately managed security groups for cluster
+and nodes. To continue to use these (and avoid recreating the cluster) set
+`legacy_security_groups = true` on the cluster module.
+
+Update terraform state:
+
+```shell
```

Comment:
It would be nice to provide this as an actual script, to ease migration and reduce errors.

Reply:
Good idea!